src/views/Ecosystem/Ecosystem.js

```diff
@@ -652,7 +652,7 @@ export default function Ecosystem() {
                   <div className="label">Announcement</div>
                   <div>
                     <a
-                      href="https://twitter.com/GMX_IO/status/1438124768033660938"
+                      href="https://twitter.com/MYC_IO/status/1438124768033660938"
```

Comment:
Could probably remove all of these as there would be no announcements from Myc account to be listed just yet